### PR TITLE
Trying out visual improvements for the tiers page. (WIP)

### PR DESCRIPTION
**Pull request by mayhem**
Check out this work in progress for improving the tiers.

### CODE REVIEW

File: metabrainz/templates/users/account-type.html [hunk 1]
```diff
@@ -42,7 +42,11 @@
       <div class="row">
         {% for tier in tiers %}
           {% if tier.primary %}
-            <div class="tier col-sm-6 col-md-4">
+            {% if loop.last %}
+                <div class="tier col-sm-6 col-md-6">
+            {% else %}
+                <div class="tier col-sm-6 col-md-4">
+            {% endif %}
```
Comment (gentlecat) [inline metabrainz/templates/users/account-type.html]: <div class="tier col-sm-6 {{ 'col-md-6' if loop.last else 'col-md-4' }}">

File: metabrainz/templates/users/account-type.html [hunk 2]
```diff
@@ -59,6 +63,15 @@
             </div>
           {% endif %}
         {% endfor %}
+        <div class="tier col-sm-6 col-md-4">
+          <div class="thumbnail">
+            <div class="caption">
+              <h3>Our proud unicorns</h3>
+              <p>Guff about unicorns.</p>
+            </div>
+            <a href="{{ url_for('users.tier', tier_id=tiers[-1].id) }}">See all our unicorns</a>.
```
Comment (gentlecat) [inline metabrainz/templates/users/account-type.html]: I think this should be a part of `caption` block above.

**Comment by gentlecat** [comment]
Here's how it looks like with [one](http://i.imgur.com/H9EeEEA.png), [two](http://i.imgur.com/I1IMWOR.png), [three](http://i.imgur.com/00OZFbr.png), [four](http://i.imgur.com/DQeSDjQ.png), and [five](http://i.imgur.com/DQeSDjQ.png) tiers.

**Comment by gentlecat** [comment]
#138, #125

**Comment by mayhem** [comment]
:+1: Please merge and deploy.

**Pull request merged**